gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/saml/SamlSsoConfig.groovy

```diff
@@ -155,6 +156,12 @@ class SamlSsoConfig extends WebSecurityConfigurerAdapter {
         def email = assertion.getSubject().nameID.value
         def roles = extractRoles(email, attributes, userAttributeMapping)
 
+        if (samlSecurityConfigProperties.requiredRoles) {
+          if (!roles.containsAll(samlSecurityConfigProperties.requiredRoles)) {
```

Comment:
I don't think it was. IIRC it was "must be a member of at least one of the specified groups"